nautobot/dcim/tables/locations.py

```diff
@@ -81,3 +82,23 @@ class Meta(BaseTable.Meta):
             "actions",
         )
         default_columns = ("pk", "name", "status", "parent", "tenant", "description", "tags", "actions")
+
+
+class FlatLocationTable(LocationTable):
```

Comment:
Do we need this class at all? Or can we just use a `LocationTable(..., hide_hierarchy_ui=True)`?